### PR TITLE
Jessica branch

### PR DESCRIPTION
**Pull request by jessicawang225**
Prototype for admin page finished:

- Logging in as a hardcoded admin user (username: admin, password: admin) and navigating to "/admin" shows three usage statistics (will work on adding more and meeting MVP requirements)
- Logging in as any other valid user will show an error telling the user they do not have access to the admin page

### CODE REVIEW

File: src/main/java/codeu/model/store/basic/UserStore.java [hunk 1]
```diff
@@ -126,5 +126,24 @@ public boolean isUserRegistered(String username) {
...
+   */
+  public int totalUsers() {
+	  return this.users.size();
+  }
+  
+  /**
+   * Returns whether or not the user is an admin
+   */
+  public boolean isAdmin(String username) {
+	  if (username.equals("admin")) {
```
Comment (ctrokid) [inline src/main/java/codeu/model/store/basic/UserStore.java]: I like how you made this a method as you've realized it's probably going to change in the future! You should just be able to shorten this to:

`return username.equals("admin");`

It will return the boolean directly for you 😋

File: src/main/java/codeu/controller/AdminServlet.java [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    }
+	
+	/**
+	   * This function fires when a user requests the /admin URL. It simply forwards the request to
+	   * admin.jsp.
+	   */
+	public void doGet(HttpServletRequest request, HttpServletResponse response)
+			throws IOException, ServletException {
+		
+		String username = (String) request.getSession().getAttribute("user");
```
Comment (ctrokid) [inline src/main/java/codeu/controller/AdminServlet.java]: There are some indentation/linting errors in this file. It's no big deal, but did you happen to figure out how to use the java linter that I linked in the chat from awhile back? https://github.com/google/google-java-format#google-java-format

I can help set this up if you need it, and I really think it will make modifying each other's code a lot easier moving forward.

**Comment by Dlindo28** [comment]
Everything looks good here. For the MVP or Final product, I think you could set up some boolean 'admin' variable for each user. Then rather than checking for the username, you could check if a user is an admin from that variable.
